nf_core/pipeline-template/nextflow.config

```diff
@@ -267,6 +260,39 @@ manifest {
     doi             = ''
 }
 
+// Nextflow plugins
+plugins {
+    id 'nf-schema@2.1.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+}
+
+includeConfig "conf/colors.config"
+
+validation {
+    parametersSchema = "${projectDir}/nextflow_schema.json"
+    help {
+        enabled = true
+        command = "nextflow run $manifest.name -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+        beforeText = """
+-${colors.dim}----------------------------------------------------${colors.reset}-
+                                        ${colors.green},--.${colors.black}/${colors.green},-.${colors.reset}
+${colors.blue}        ___     __   __   __   ___     ${colors.green}/,-._.--~\'${colors.reset}
+${colors.blue}  |\\ | |__  __ /  ` /  \\ |__) |__         ${colors.yellow}}  {${colors.reset}
+${colors.blue}  | \\| |       \\__, \\__/ |  \\ |___     ${colors.green}\\`-._,-`-,${colors.reset}
+                                        ${colors.green}`._,._,\'${colors.reset}
+${colors.purple}  ${manifest.name} ${manifest.version}${colors.reset}
+-${colors.dim}----------------------------------------------------${colors.reset}-
+"""
+        afterText = """${manifest.doi ? "* The pipeline\n" : ""}${manifest.doi.tokenize(",").collect { "  https://doi.org/${it.trim().replace('https://doi.org/','')}"}.join("\n")}${manifest.doi ? "\n" : ""}
+* The nf-core framework 
+  https://doi.org/10.1038/s41587-020-0439-x
+
+* Software dependencies
+  https://github.com/${manifest.name}/blob/master/CITATIONS.md
+"""
```

Comment:
should we lint for this in nf-core pipelines?

Reply:
Good idea, but I want to implement this in a better way as it's soooo ugly right now, I'm just not sure how. 

I'll also add a chrome color filter to nf-schema if `validation.monochromeLogs` is set to `true`. This should simplify the configs a lot already (We can hardcode the monochrome values in the message together with the pipeline name and version, same thing for the citations) I'd love to have these values as plain strings without having to use any external variables

Reply:
I think the current way will be the best I can do. This will also stay supported in Nextflow in the future